docs/BR.md

```diff
@@ -1055,7 +1055,8 @@ The CA SHALL revoke a Certificate within 24 hours if one or more of the followin
 1. The Subscriber requests in writing that the CA revoke the Certificate;
 2. The Subscriber notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization;
 3. The CA obtains evidence that the Subscriber's Private Key corresponding to the Public Key in the Certificate suffered a Key Compromise;
-4. The CA obtains evidence that the validation of domain authorization or control for any Fully-Qualified Domain Name or IP address in the Certificate should not be relied upon.
+4. The CA is made aware of a demonstrated or proven method that can easily compute the Subscriber's Private Key based on the Public Key in the Certificate (such as a Debian weak key, see https://wiki.debian.org/SSLkeys);
```

Comment:
While it would appear this is moving from 5 day to 24 hour, this is really just a clarified restatement of 3 (that is, demonstration of easy, arbitrary recovery IS a Key Compromise)

Reply:
IMO, this change along with the 6.1.1.3 and 9.6.3(8) proposals are more than a clarification and should be combined into a separate 'revocation improvements' ballot.

Reply:
Thanks. I appreciate the feedback, although I (clearly :P) disagree. This was covered on the posting at https://cabforum.org/pipermail/servercert-wg/2020-April/001815.html about how this is:

> "Clarify an existing /implicit/ requirement by making it /explicit", and
then secondarily "Clarify the logical consequences of an existing
requirement".

I can understand a desire to have a ballot that is only "uncontroversial" changes, but the value of that is largely dependent on the urgency of fixing the issues. Given that these areas have caused significant confusion and issues, I think they're important to clarify, and I don't think the remaining changes are particularly harmed by having that discussion sooner than later. So if you disagree with the substance, such as disagree that they logically follow or that they're reasonable expectations, and not just the process points, I think it'd be useful to have on the list.

Reply:
I'm concerned that this wording is still confusing, because a Debian weak key isn't a "method", as such, unless you want to get all reductionist and say that "looking up the public key in a hash table" is a "method".  If you asked me "give me an example of a method to easily compute a private key based on a public key", I'd think "RoCA" long before I thought "Debian weak key".

That being said, I don't have an immediate suggestion as to better wording that isn't either as specific as "don't issue certificates that use Debian weak keys", or so broad as to be meaningless, like "any key which is included in a widely-publicised list of unsuitable keys".  I am inclined to think that a better description of Debian weak keys is actually the definition still under the "5 day" time period: "the specific method used to generate the Private Key was flawed".  I think we can all agree that seeding your RNG with a 15 bit number is a flawed method with which to generate a private key...

Reply:
@mpalmer Thanks for your feedback. I don’t think it’s as black or white as you suggest, however. Computing the private key by looking up in a table of known weak public keys is a method, as would generating a set of known weak keys by reproducing the code in question for its permutations.